user_stories.md

```diff
@@ -0,0 +1,11 @@
+# Пользовательские истории.
```

Comment:
You missed motivational part of your stories. Add them, please.

Reply:
Our collective mind interpreted it as "add motivation to the stories where it's missed" (e.g. user story no.8 about statistics), but if it's wrong, could you, please, explain what you meant?